NuRadioMC/utilities/medium.py

```diff
@@ -1,112 +1,237 @@
+        """
+        returns the gradient of index of refraction at position
+        Overwrites function of the mother class
 
-class mooresbay_simple(medium):
+        Parameters
+        ----------
+        position: 3dim np.array
+                    point
 
-    # from https://doi.org/10.1088/1475-7516/2018/07/055 MB1 model
-    def __init__(self):
-        self.n_ice = 1.78
-        self.z_0 = 34.5 * units.m
-        self.delta_n = 0.46
-        self.reflection = -576 * units.m  # from https://doi.org/10.3189/2015JoG14J214
-        self.reflection_coefficient = 0.82  # from https://doi.org/10.3189/2015JoG14J214
-        self.reflection_phase_shift = 180 * units.deg
+        Returns
+        -------
+        n_nabla:    (3,) np.array
+                    gradient of index of refraction at the point
+        """
+        pos = RP.Vector3d(*(position*RP.meter/units.meter))
+        return self._scalarfield.getGradient(pos)  * (1/(units.meter/RP.meter))
 
+    
+    def get_ice_model_radiopropa(self,discontinuity=False):
+        """
+        Returns an object holding the radiopropa scalarfield and necessary radiopropa moduldes 
+        that define the medium in radiopropa. It uses the parameters of the medium object to 
+        contruct some modules, like a discontinuity object for the air boundary. Additional modules
+        can be added in this function
+        
+        Overwrites function of the mother class
+
+        Returns
+        -------
+        ice:    RadioPropaIceWrapper
+                object holding the radiopropa scalarfield and modules
+        """
+        ice = RadioPropaIceWrapper(self,self._scalarfield)
+        if discontinuity == True:
+            firn_boundary_pos = RP.Vector3d(0,0,self.z_firn*(RP.meter/units.meter))
+            step = RP.Vector3d(0,0,1e-9*RP.meter)
+            firn_boundary = RP.Discontinuity(RP.Plane(firn_boundary_pos, RP.Vector3d(0,0,1)), 
+                            self._scalarfield.getValue(firn_boundary_pos-step),
+                            self._scalarfield.getValue(firn_boundary_pos+step))
+            ice.add_module('firn boudary',firn_boundary)
+        return ice
 
-class mooresbay_simple_2(medium):
 
-    # from https://doi.org/10.1088/1475-7516/2018/07/055 MB2 model
-    def __init__(self):
-        self.n_ice = 1.78
-        self.z_0 = 37 * units.m
-        self.delta_n = 0.481
-        self.reflection = -576 * units.m  # from https://doi.org/10.3189/2015JoG14J214
-        self.reflection_coefficient = 0.82  # from https://doi.org/10.3189/2015JoG14J214
-        self.reflection_phase_shift = 180 * units.deg
 
 
-class greenland_simple(medium):
 
-    # from C. Deaconu, fit to data from Hawley '08, Alley '88
-    # rho(z) = 917 - 602 * exp (-z/37.25), using n = 1 + 0.78 rho(z)/rho_0
-    def __init__(self):
-        self.n_ice = 1.78
-        self.z_0 = 37.25 * units.m
-        self.delta_n = 0.51
+def get_ice_model(name):
```

Comment:
I did it this way because otherwise you always had to had new ice_models to the list which might become very long over time. Since it is only a function to get the class with that name I thought it might be okay but I'll update the docstring so it become very clear what the function does